simplified-app-shared/src/main/java/org/nypl/simplified/app/catalog/CatalogFeedActivity.java

```diff
@@ -678,18 +687,21 @@ private void onCreateOptionsMenuSearchItem(
       final Some<FeedSearchType> search_some =
         (Some<FeedSearchType>) search_opt;
 
-      final SearchView sv = (SearchView) search_item.getActionView();
-      sv.setSubmitButtonEnabled(true);
+      this.search_view = (SearchView) search_item.getActionView();
+      this.search_view.setSubmitButtonEnabled(true);
+      this.search_view.setIconifiedByDefault(false);
+      search_item.setShowAsActionFlags(MenuItem.SHOW_AS_ACTION_ALWAYS);
+      search_item.expandActionView();
 
       /**
        * Set some placeholder text
        */
 
       final CatalogFeedArgumentsType args = this.getArguments();
-      if (this.getUpStack().isEmpty()) {
-        sv.setQueryHint("Search");
-      } else {
-        sv.setQueryHint("Search " + args.getTitle());
+      this.search_view.setQueryHint("Search " + this.feed.getFeedTitle());
```

Comment:
yes, it could, will add this as part of the next branch where i have added "all" the missing localized strings